Lib/idlelib/autocomplete.py

```diff
@@ -175,6 +176,7 @@ def fetch_completions(self, what, mode):
                     namespace = {**__main__.__builtins__.__dict__,
                                  **__main__.__dict__}
                     bigl = eval("dir()", namespace)
+                    bigl.extend(keyword.kwlist)
```

Comment:
This looks good, but `True` is on the list twice, so it looks like `bigl` needs to be dedupped.

Reply:
Ditto for None and False.  Thanks for catching this.  Revision excludes None and True by length and False by equality and includes a new test of each.